M2/Macaulay2/packages/Macaulay2Doc/debugging.m2

```diff
@@ -50,7 +50,7 @@ document {
      code f
      f 0
      return
-     disassemble current
+     current
```

Comment:
Did you mean `pseudocode current` here?

Reply:
no I think it's correct as is. current is already pseudocode, it's just that now there's no need to `disassemble` it.